app/settings.py

```diff
@@ -153,3 +153,5 @@ def utcoffset_or_fail(date_value, key):
 PRINT_STYLE_SHEET_FILE_PATH = os.getenv(
     "PRINT_STYLE_SHEET_FILEPATH", "templates/assets/styles"
 )
+
+ONS_URL = os.getenv("ONS_URL", "https://www.ons.gov.uk")
```

Comment:
Perhaps  `ONS_BASE_URL` ?

Reply:
I prefer `ONS_URL` as we already have `base_url` in this namespace, so I would just keep it as it is for clarity.